### PR TITLE
Added advantages & disadvantages of using ajax

### PR DESCRIPTION
**Pull request by gauravmuk**
This is one of my favourite questions that I ask myself. This justifies if a person really knows what are the actual powers of AJAX.

From here, we can move towards JSONP question since for someone who knows the limitations of ajax would definitely know something about or at-least heard about JSONP.

### CODE REVIEW

**Comment by phanan** [comment]
I think this question is more or less part of the previous one: 

> Explain AJAX in as much detail as possible.

**Comment by gauravmuk** [comment]
But this gives a direction to the question. Somebody might be able to tell
you how it is useful but might not think of the shortcomings.

**Comment by phanan** [comment]
I still think one question should be enough. Maybe the current one, maybe yours, maybe a merged version, but not both.

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ This file contains a number of front-end interview questions that can be used wh
 * When would you use `document.write()`?
 * What's the difference between feature detection, feature inference, and using the UA string?
 * Explain AJAX in as much detail as possible.
+* What are the advantages and disadvantages of using AJAX?
 * Explain how JSONP works (and how it's not really AJAX).
```
Comment (arthurvr) [inline README.md]: I'd prefer *Ajax* instead of *AJAX*.

Comment (appleboy) [inline README.md]: agree with @arthurvr 

ref: https://en.wikipedia.org/wiki/Ajax_(programming)

**Comment by gauravmuk** [comment]
@arthurvr changed the same in the PR.

**Pull request merged**